From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Pardini <ricardo@pardini.net>
Date: Mon, 5 Jan 2026 16:32:45 +0100
Subject: GENIO: mt8195: fdt_addr_r/kernel_addr_r/ramdisk_addr_r for large
 kernels/initrds

General purpose OS's like Armbian have some huge initrds and kernels, make space.

Signed-off-by: Ricardo Pardini <ricardo@pardini.net>
---
 include/configs/mt8195.h | 12 +++++-----
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/include/configs/mt8195.h b/include/configs/mt8195.h
index 111111111111..222222222222 100644
--- a/include/configs/mt8195.h
+++ b/include/configs/mt8195.h
@@ -133,11 +133,11 @@
 #if !IS_ENABLED(CONFIG_SPI_FLASH)
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"scriptaddr=0x40000000\0" \
-	"fdt_addr_r=0x44000000\0" \
+	"fdt_addr_r=0x56000000\0" \
 	"fdtoverlay_addr_r=0x44c00000\0" \
 	"fdt_resize=0x3000\0" \
-	"kernel_addr_r=0x45000000\0" \
-	"ramdisk_addr_r=0x49000000\0" \
+	"kernel_addr_r=0x40000000\0" \
+	"ramdisk_addr_r=0x64000000\0" \
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE ".dtb\0" \
 	"splashimage=" __stringify(CONFIG_SYS_LOAD_ADDR) "\0" \
 	"splashsource=mmc_fs\0" \
@@ -148,11 +148,11 @@
 #else
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"scriptaddr=0x40000000\0" \
-	"fdt_addr_r=0x44000000\0" \
+	"fdt_addr_r=0x56000000\0" \
 	"fdtoverlay_addr_r=0x44c00000\0" \
 	"fdt_resize=0x3000\0" \
-	"kernel_addr_r=0x45000000\0" \
-	"ramdisk_addr_r=0x49000000\0" \
+	"kernel_addr_r=0x40000000\0" \
+	"ramdisk_addr_r=0x64000000\0" \
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE ".dtb\0" \
 	"splashimage=" __stringify(CONFIG_SYS_LOAD_ADDR) "\0" \
 	"splashsource=sf\0" \
-- 
Armbian

